views/imagestream-body.html

```diff
@@ -36,6 +36,8 @@
     <dd ng-if-end><tt>{{imagestream.spec.dockerImageRepository}}</tt></dd>
     <dt>Pulling repository</dt>
     <dd><tt>{{imagestream.status.dockerImageRepository}}</tt></dd>
+    <dt ng-if-start="imagestream.status.publicDockerImageRepository">Public pulling repo</dt>
```

Comment:
Missing _translate_.
Can you link docs for what this attribute is to confirm nomenclature?
Thanks!

Reply:
Not sure if we have public-facing docs yet, but here are godocs from the PR that introduced these fields: https://github.com/openshift/origin/pull/15853/files#diff-3788994f990386b63387ece5c3b5dc33 and https://github.com/openshift/origin/pull/15853/files#diff-1a35f433d29ab01f47e5563ac4933758R136